### PR TITLE
Mohammadh/ Add derived to help centre

### PR DESCRIPTION
**Pull request by mohammad-hashemi-deriv**
Changes:

-   ...

## Type of change

-   [ ] Bug fix
-   [ ] New feature
-   [x] Update feature
-   [ ] Refactor code
-   [ ] Translation to code
-   [ ] Translation to crowdin
-   [ ] Script configuration
-   [ ] Improve performance
-   [ ] Style only
-   [ ] Dependency update
-   [ ] Documentation update
-   [ ] Release

### CODE REVIEW

File: src/pages/help-centre/_article-component.tsx [hunk 1]
```diff
@@ -140,33 +140,38 @@ const ArticleComponent = ({
...
                     const can_expand = item.articles.length > 3
                     const should_show_expand = !category_is_expanded && can_expand && idxb === 3
                     const should_show_collapse =
                         category_is_expanded && can_expand && idxb === item.articles.length - 1
-                    const title_type = is_eu_country && ar.title_eu ? ar.title_eu : ar.title
-
-                    const label_type = is_eu_country && ar.label_eu ? ar.label_eu : ar.label
 
                     return (
                         <ListNoBullets key={idxb}>
```
Comment (michio-rework) [inline src/pages/help-centre/_article-component.tsx:165]: please don't use map `index` as key :pray:

File: src/pages/help-centre/dmt5.tsx [hunk 2]
```diff
@@ -18,87 +18,182 @@ const StyledListItem = styled.li<{ marginTop: string }>`
...
+            <StyledText>
+                <Localize
+                    translate_text="<0>DTrader</0> offers multipliers trading on a range of underlying assets such as forex, cryptocurrencies, and derived. This platform allows you to open multipliers trades that offer the opportunity to multiply potential profit without risking more than your stake. You may find it more intuitive if you're new to the trading world."
+                    components={[
+                        <ExternalLink
+                            to={'/dtrader/'}
+                            external
+                            target="_blank"
+                            rel="noopener noreferrer"
+                            key={0}
```
Comment (michio-rework) [inline src/pages/help-centre/dmt5.tsx:175]: do we need `key` here?

File: src/pages/help-centre/dmt5.tsx [hunk 3]
```diff
@@ -18,87 +18,182 @@ const StyledListItem = styled.li<{ marginTop: string }>`
...
+            <StyledText>
+                <Localize
+                    translate_text="<0>Deriv MT5</0> offers CFD trading on a similar range of assets, where you're able to open positions with leverage and your potential profit is only known when you close your positions. It's riskier than multipliers trading because while you may potentially gain a lot if you win, you may also lose a lot if you don't. Deriv MT5 is popular among our traders who enjoy the risks as part of the excitement of CFD trading."
+                    components={[
+                        <ExternalLink
+                            to={'/dmt5/'}
+                            external
+                            target="_blank"
+                            rel="noopener noreferrer"
+                            key={0}
```
Comment (michio-rework) [inline src/pages/help-centre/dmt5.tsx:189]: do we need `key` here?

File: src/pages/help-centre/trading.tsx [hunk 4]
```diff
@@ -40,6 +40,54 @@ const Forex = ({ text }: ArticleProps) => (
...
+            <Localize
+                translate_text="Available 24/7, our synthetic indices emulate price movements of real-world markets with varying levels of volatility. As they aren't based on actual underlying assets, they are unaffected by real-world market events. The pricing of our synthetic indices is backed by algorithms that are audited for fairness by an independent third party. Read <0>this article</0> to learn more about trading synthetic indices on Deriv."
+                components={[
+                    <StyledLink
+                        to={'/academy/blog/posts/an-introduction-to-synthetic-indices-trading/'}
+                        target="_blank"
+                        external
+                        weight="bold"
+                        rel="noopener noreferrer"
+                        key={0}
```
Comment (michio-rework) [inline src/pages/help-centre/trading.tsx:61]: do we need `key` here?

File: src/pages/help-centre/trading.tsx [hunk 5]
```diff
@@ -40,6 +40,54 @@ const Forex = ({ text }: ArticleProps) => (
...
+            <Localize
+                translate_text="Due to regulatory requirements, synthetic indices are unavailable in some countries. Refer to ‘Product offering’ in our <0>terms of use</0> for more info."
+                components={[
+                    <StyledLink
+                        to={'/tnc/general-terms.pdf'}
+                        target="_blank"
+                        external
+                        weight="bold"
+                        rel="noopener noreferrer"
+                        key={0}
```
Comment (michio-rework) [inline src/pages/help-centre/trading.tsx:83]: do we need `key` here?

**Review by michio-rework — CHANGES_REQUESTED** [review]

File: src/pages/help-centre/dmt5.tsx (hunk 2 above)
Comment (mohammad-rework) [reply src/pages/help-centre/dmt5.tsx:175]: `key` is always used in `<Localize components={[]} />`. It seems that components property only accepts array of component and it expects `key` for each component. Do we need to refactor and remove `key` everywhere we have only one item in the array?

Comment (michio-rework) [reply src/pages/help-centre/dmt5.tsx:175]: Oh Ok I see, I didn't know that, then it's ok, thanks @mohammad-rework :pray:

**Review by michio-rework — DISMISSED** [review]

**Pull request closed without merging**